data/bangs.json

```diff
@@ -51026,14 +51026,6 @@
     "c": "Online Services",
     "sc": "Maps"
   },
-  {
-    "s": "Kicktraq",
-    "d": "www.kicktraq.com",
-    "t": "kt",
```

Comment:
You can just move the kicktraq bang to `kicktraq` instead if removing it

Reply:
Oh, there's already a !kicktraq bang for kicktraq